templates/manila/config/manila.conf

```diff
@@ -1,3437 +1,73 @@
 [DEFAULT]
+state_path=/var/lib/manila
+enabled_share_backends = alpha
+host=hostgroup
+storage_availability_zone=nova
+default_share_type=default
+rootwrap_config=/etc/manila/rootwrap.conf
+auth_strategy=keystone
+network_api_class=manila.network.neutron.neutron_network_plugin.NeutronNetworkPlugin
+network_plugin_ipv4_enabled=True
+network_plugin_ipv6_enabled=False
```

Comment:
not needed; not harmful for testing though

Reply:
removing!